### PR TITLE
Fix pipeline hanging on error

### LINKED ISSUE #466: Face ReID demo hangs on error
It can be reproduced under the conditions described in #465 or under the conditions described below.

Use, for example, `./scripts/run_module.py` to start the demo with `module_performance.yml` config. 

To reproduce, modify the face_preproc.py, and raise an exception. If an exception occurs, the module will not terminate after the source has been processed and the module container will go to the "unhealthy" status after some time.

### PR DESCRIPTION
**Pull request by tomskikh**

### CODE REVIEW

File: savant/entrypoint/main.py [hunk 1]
```diff
@@ -89,3 +89,7 @@ def main(config_file_path: str):
                 os._exit(1)  # pylint: disable=protected-access
     except Exception as exc:  # pylint: disable=broad-except
         logger.error(exc, exc_info=True)
+        exit(1)
```
Comment (denisvmedyantsev) [inline savant/entrypoint/main.py:92]: What is the reason for `exit` here? `main` is the only function in entrypoint and there it's your final row

File: savant/entrypoint/main.py [hunk 2]
```diff
@@ -89,3 +89,7 @@ def main(config_file_path: str):
...
+
+    if runner.error is not None:
```
Comment (denisvmedyantsev) [inline savant/entrypoint/main.py:94]: `runner` is out of its context?

**Review by denisvmedyantsev — CHANGES_REQUESTED** [review]

File: savant/entrypoint/main.py (hunk 1 above)
Comment (tomskikh) [reply savant/entrypoint/main.py:92]: `exit` here is needed to stop the module with non-zero exit code.

File: savant/entrypoint/main.py (hunk 2 above)
Comment (tomskikh) [reply savant/entrypoint/main.py:94]: We don't need context here.

**Review by denisvmedyantsev — APPROVED** [review]

**Pull request merged**